ChangeLog.d/fix-time-format-pre-2000.txt

```diff
@@ -0,0 +1,2 @@
+Bugfix
+   * Fix X509 time format tags for dates before 1/1/2000.
```

Comment:
minor: this should probably give a bit more detail than just "fix", and since it closes an issue, it should mention the issue number, e.g.
```suggestion
   * Encode X.509 dates before 1/1/2000 as UTCTime rather than GeneralizedTime.
     Fixes #5645.
```